core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7546,6 +7546,12 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.SERVER)
           .build();
+  public static final PropertyKey HADOOP_CHECKSUM_COMBINE_MODE =
+      booleanBuilder(Name.HADOOP_CHECKSUM_COMBINE_MODE)
+          .setDescription("File Checksum combine mode.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.CLIENT)
```

Comment:
is "client" the proper scope for the property?

Reply:
at Hadoop side it's client property so I would keep the same behavior